internal/middleware/prometheus.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-var buckets = []float64{100, 200, 500, 5000}
+var buckets = []float64{100, 200, 500, 1000, 2000, 5000}
```

Comment:
Would 750 make sense here? given we are slightly over 500 on the offending endpoint, I guess it might be nice to have bit more granuality there, right?